senior-knights/course-schedulizer
# read xlsx schedules

Adds the ability to upload `.xlsx` (Excel) files to the course schedulizer. 

To Test
- `npm i && npm start`
- Try uploading the .xlsx math schedule from `csv/`
- Next (perhaps in a new tab), upload the .csv version of the math schedule and notice the same schedule is uploaded.

I am proposing this change since I will like it will make the app easier to use for department chairs who might have trouble converting from one file format to another. 

The code is a little ugly, so I would be open to suggestions to make it look nicer. 

closes #87
